clm_ros_wrapper/VectorWithCertainty[] vectors
